cmd/qat_plugin/README.md

```diff
@@ -187,6 +187,19 @@ For example, `qat.intel.com/generic: <number of devices>` for a container reques
 
 For a DPDK-based workload, you may need to add hugepage request and limit.
 
+##### NOTES
+1. In the below file, modify line 21 and 26 to reflect the custom resources in your worker node. cy=crypto and dc=compression
+It should look like below based on your scenario
+
+qat.intel.com/cy: '4'
+or
+qat.intel.com/dc: '4'
+https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/deployments/qat_dpdk_app/base/crypto-perf-dpdk-pod-requesting-qat.yaml
+
+Also, you can check by describing the workr node and verify the Allocatable resources.
+![Alt text](worker-node-describe.png)
```

Comment:
add this as as text output in "verify plugin registration" below the existing sample.